coffeecard/CoffeeCard.Models/DataTransferObjects/v2/Ticket/TicketResponse.cs

```diff
@@ -57,5 +57,14 @@ public class TicketResponse
         /// <example>Coffee</example>
         [Required]
         public string ProductName { get; set; } = string.Empty;
+
+        /// <summary>
+        /// The name of the menu item that this ticket was used on
+        /// </summary>
+        /// <value>Menu item Id</value>
+        /// <example>Cappuccino</example>
+        /// <remarks>Null if ticket was not used on a menu item</remarks>
+        [JsonProperty(Required = Required.Default)]
```

Comment:
I'm not sure we need this annotation but maybe I overlooked something - why did you add it?

Reply:
Oversight - the Default value is, well, the default anyway. At some point it was set to DisallowNull instead ("The property is not required but it cannot be a null value.") I'm still not sure if that is needed though, what do you think?

Reply:
Removed in 7dd4783a594872f97a7592461f2857f59403ead5